From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@matt1883__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mahi). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

begoldsm commented:
Acknowledged. I'm looking this over now.

PRmerger commented:
@matt1883 : Thanks for your contribution to the Azure documentation! The author, @nitinme, has been notified to review your proposed change.

begoldsm commented:
Looks good to me!

matt1883 commented:
Thanks, @begoldsm!

@mumian, since @nitinme is not available today, could you review when possible, please?

mumian commented:
#sing-off

mumian commented:
Nitin is OOF.  I have reviewed the changes. #sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

matt1883 commented:
@cjgronlund FYI this is ready to merge

cjgronlund commented:
@mumian @matt1883 
This article is now in the private repo in this PR: https://github.com/Azure/azure-content-pr/pull/29370

Please sign-off there. 

This PR can be closed.

MRayermannMSFT committed:
Update Storage Explorer docs for GA

Initial work for updating docs for GA

Improve arolinx score for the Storage Explorer troubleshooting guiide

Remove unused image

Improve arolinx score for connect storage explorer to azure stack

Update contents of connect storage explorer to azure stack

Update contents of connect storage explorer to azure stack pt2

Specify bitness for ubuntu versions

Add line about root and intermediate certificates, closes #5057

Fix misspelling

Add note about lib secret

Add shortcut keys

Update table headers

Comment about needing to do settings on OS level as well

Acrolinx

Move some things around

JasonWHowell committed:
Update stream-analytics-define-outputs.md

Commit:
test-commit

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-405

Commit:
Revert "Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr"

This reverts commit b1b9ea135551773f77ba486838bab338d83e87ef, reversing
changes made to 03dddbc57dab9a0b63ec73e7fad9a1adba52b379.

Commit:
Revert "Revert "Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr""

This reverts commit d4f5fb2536c7ab1fe4c0ccb6b5538d729e02e43c.

Commit:
Revert "test-commit"

This reverts commit 03dddbc57dab9a0b63ec73e7fad9a1adba52b379.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-405

JasonWHowell committed:
Adding recovery information

JasonWHowell committed:
Adding recovery information

JasonWHowell committed:
Adding recovery information

Commit:
Task 405: Add Verizon-specific header info

JasonWHowell committed:
Adding recovery information

JasonWHowell committed:
Adding recovery information

JasonWHowell committed:
Update stream-analytics-concepts-checkpoint-replay.md

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-405

 enter a commit message to explain why this merge is necessary,

Commit:
Task 405: Format edits

chetansriv committed:
FluxxLabs tutorial update

chetansriv committed:
Merge branch 'master' into fluxxlabs

JasonWHowell committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into stream-analytics

JasonWHowell committed:
Merge branch 'stream-analytics' of https://github.com/jasonwhowell/azure-docs-pr into stream-analytics

Commit:
Task 405: Edits

jeahyoun committed:
Create test context file.

JasonWHowell committed:
Dividing into three articles

JasonWHowell committed:
Dividing into three articles

JasonWHowell committed:
Dividing into three articles

JasonWHowell committed:
Dividing into three articles

Commit:
redirect develop and deploy doc

Commit:
update toc

cephalin committed:
reshoot screenshots

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-405

Commit:
Task 405: Split X-EC-Debug headers into new topic

Commit:
Task 405: Edits

Commit:
Task 405: Edits

msvijayn committed:
Update ARM templates for Alerts with ActionGroup

msvijayn committed:
acrolinux update

Commit:
Update .openpublishing.redirection.json

v-geberr committed:
phrase list clarification

KumudD committed:
new article

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dnsalerts0415

KumudD committed:
fixed Acrolinx issues

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dnsalerts0415

v-geberr committed:
HTTP status codes

ArLucaID committed:
Introducing MSI over IMDS in public docs

ArLucaID committed:
Add files via upload

ArLucaID committed:
Delete VM MSI IMDS diagram.png

ArLucaID committed:
Add files via upload

ArLucaID committed:
Update overview.md

Commit:
Task 405: Fix link

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-405

ArLucaID committed:
Update tutorial-windows-vm-access-arm.md

douglaslMS committed:
New article about SSIS Activity in ADF v2.

ArLucaID committed:
Update tutorial-linux-vm-access-arm.md

ArLucaID committed:
Update how-to-use-vm-sign-in.md

douglaslMS committed:
Added new article to TOC.

KumudD committed:
added media files

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dnsalerts0415

linda33wj committed:
Fix SAP C4C copy source type in sample

KumudD committed:
resized art files

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dnsalerts0415

douglaslMS committed:
Made some Acrolinx fixes.

ArLucaID committed:
Update how-to-use-vm-token.md

KumudD committed:
fixed links

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dnsalerts0415

ArLucaID committed:
Update known-issues.md

ArLucaID committed:
Update tutorial-linux-vm-access-cosmos-db.md

ArLucaID committed:
Update tutorial-linux-vm-access-datalake.md

BrentL-Collabera committed:
Add files via upload

ArLucaID committed:
Update tutorial-linux-vm-access-nonaad.md

cynthn committed:
Update virtual-machines-common-sizes-general.md

BrentL-Collabera committed:
Add files via upload

ArLucaID committed:
Update tutorial-linux-vm-access-storage-access-key.md

ArLucaID committed:
Update tutorial-linux-vm-access-storage-sas.md

KumudD committed:
formatted images

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dnsalerts0415

ArLucaID committed:
Update tutorial-linux-vm-access-storage.md

ArLucaID committed:
Update tutorial-vm-windows-access-storage.md

ArLucaID committed:
Update tutorial-windows-vm-access-cosmos-db.md

ArLucaID committed:
Update tutorial-windows-vm-access-datalake.md

ArLucaID committed:
Update tutorial-windows-vm-access-nonaad.md

wesmc7777 committed:
Merge pull request #8 from MicrosoftDocs/master

merge with master

ArLucaID committed:
Update tutorial-windows-vm-access-sql.md

ArLucaID committed:
Update tutorial-windows-vm-access-storage-sas.md

ArLucaID committed:
Update tutorial-windows-vm-access-storage.md

ArLucaID committed:
Update tutorial-windows-vm-access-storage.md

BrentL-Collabera committed:
Merge branch 'master' into New_Azure_Marketplace_Docs

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into routing2

mrbullwinkle committed:
image update to address github issue 7077

BrentL-Collabera committed:
Update marketplace-grow-your-business-with-appsource.md

Fixed several Acrolinx errors

Commit:
Update analysis-services-scale-out.md

BrentL-Collabera committed:
Update marketplace-grow-your-business-with-appsource.md

Fix broken link.

Commit:
Update analysis-services-scale-out.md

BrentL-Collabera committed:
Update marketplace-grow-your-business-with-appsource.md

Attempting to fix bad link.

Commit:
Update analysis-services-scale-out.md

hpsin committed:
Minor updates to page.

Commit:
update repo link

hpsin committed:
Update active-directory-configurable-token-lifetimes.md

KumudD committed:
fixed image

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dnsalerts0415

ArLucaID committed:
Add files via upload

JasonWHowell committed:
Update stream-analytics-define-outputs.md

KumudD committed:
tweak

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dnsalerts0415

BrentL-Collabera committed:
Update marketplace-grow-your-business-with-appsource.md

Added back the required metadata section.

femsulu committed:
(Forums Team) Fix broken link: #7024

Using the device SDKs -> C# link is broken. This might be the valid link but not sure...
https://github.com/Azure/azure-iot-sdk-csharp/tree/master/iothub/device/samples

Commit:
PR comments

KumudD committed:
blurred name

Commit:
Merge pull request #37432 from JasonWHowell/patch-2

Adding Partition info - Update stream-analytics-define-outputs.md

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dnsalerts0415

Commit:
Merge pull request #38002 from femsulu/patch-11

(Forums Team) Fix broken link: #7024

rmca14 committed:
Merge pull request #37991 from cherylmc/routing2

Routing2

wesmc7777 committed:
fixes Microsoft/azure-docs#7171

wesmc7777 committed:
fixes MicrosoftDocs/azure-docs#7171

KumudD committed:
fixed formatting

wesmc7777 committed:
Merge branch 'redis_updates' of https://github.com/wesmc7777/azure-docs-pr into redis_updates

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dnsalerts0415

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#6730

JasonWHowell committed:
Update stream-analytics-define-outputs.md

garyericson committed:
Fixed invalid ms.date formats

wesmc7777 committed:
link fix

tfitzmac committed:
clarified delivery

GitHubber17 committed:
Update dns-alerts-metrics.md

garyericson committed:
Fixed extra semi-colon in ms.author value #1

tfitzmac committed:
clarified topic design

wesmc7777 committed:
group VNET FAQs as H3s

GitHubber17 committed:
Merge pull request #37989 from mrbullwinkle/mrb_04_16_2018_dynamics

image update to address github issue 7077

tfitzmac committed:
fixed spacing

wesmc7777 committed:
format connectionstring examples better

Commit:
Merge pull request #38019 from JasonWHowell/patch-1

Update stream-analytics-define-outputs.md

Commit:
renamed png

Commit:
renamed png

BrentL-Collabera committed:
Update TOC.md

Fixed a line break typo.

garyericson committed:
Fixed extra semi-colon in ms.author value #2

rmca14 committed:
Merge pull request #37956 from arluca/patch-8

Introducing MSI over IMDS in public docs

Commit:
Merge pull request #38020 from tfitzmac/0416delivery

fixes MicrosoftDocs/azure-docs#6730

PRMerger10 committed:
Merge pull request #38014 from wesmc7777/redis_updates

fixes MicrosoftDocs/azure-docs#7171

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#6918

PRMerger12 committed:
Merge pull request #38024 from tfitzmac/0416concepts

clarified topic design

BryanLa committed:
Clarify CMake command and attestation mechanism

BryanLa committed:
Update quick-create-simulated-device-x509.md

garyericson committed:
Fixed extra semi-colon in ms.author value #3

PRMerger16 committed:
Merge pull request #38033 from tfitzmac/0416error

fixes MicrosoftDocs/azure-docs#6918

garyericson committed:
Fixed extra semi-colon in ms.author value #4

Commit:
Update to PowerShell section.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aas0416

garyericson committed:
Fixed extra semi-colon in ms.author value #5

PRMerger18 committed:
Merge pull request #38036 from BryanLa/patch-22

Clarify CMake command and attestation mechanism

PRMerger19 committed:
Merge pull request #38038 from BryanLa/patch-23

Clarify CMake command and attestation mechanism

PRMerger20 committed:
Merge pull request #38044 from Minewiskan/aas0416

Aas0416

GitHubber17 committed:
Merge pull request #37942 from KumudD/dnsalerts0415

DNS Metrics and Alerts

PRMerger20 committed:
Merge pull request #38022 from garyericson/04-16-fix-semicolon-in-ms-author

Fixed extra semi-colon in ms.author value #1

Commit:
Merge pull request #38027 from garyericson/04-16-2-fix-semicolon-in-ms-author

Fixed extra semi-colon in ms.author value #2

Commit:
Merge pull request #38043 from garyericson/04-16-04-fix-semicolon-in-ms-author

Fixed extra semi-colon in ms.author value #4

JasonWHowell committed:
Update stream-analytics-introduction.md

BrentL-Collabera committed:
Update marketplace-grow-your-business-with-appsource.md

Changed title metadata to include Azure.

BrentL-Collabera committed:
Update marketplace-what-is-appsource.md

Updated title metadata to include Azure.
Adjusted headings.

Commit:
Merge pull request #38042 from garyericson/04-16-03-fix-semicolon-in-ms-author

Fixed extra semi-colon in ms.author value #3

Commit:
Merge pull request #38021 from garyericson/04-16-fix-ms-date-2

Fixed invalid ms.date formats

PRMerger10 committed:
Merge pull request #38045 from garyericson/04-16-05-fix-semicolon-in-ms-author

Fixed extra semi-colon in ms.author value #5

gatneil committed:
fixed portal snapshot

Commit:
Adding info for IoT Hub

Commit:
small edit

rimman committed:
Corrected for SQL API

JasonWHowell committed:
Update stream-analytics-concepts-checkpoint-replay.md

Commit:
Task 405: Incorporate review comments, remove HTTP overview article

PRMerger20 committed:
Merge pull request #38053 from cynthn/master

Update virtual-machines-common-sizes-general.md

PRMerger15 committed:
Merge pull request #38051 from JSeb225/patch-26

Adding info for IoT Hub - Fixes MicrosoftDocs/azure-docs#6828

JasonWHowell committed:
Update stream-analytics-machine-learning-integration-tutorial.md

JasonWHowell committed:
Update stream-analytics-machine-learning-integration-tutorial.md

MGoedtel committed:
updated articles per PM feedback

MGoedtel committed:
updated H2 heading

Commit:
Reassigning Davids Windows topics

Commit:
Merge pull request #38061 from JasonWHowell/patch-2

Update stream-analytics-machine-learning-integration-tutorial.md

Commit:
Task 405: Improve Acrolinx scores

Commit:
quickfix08

GitHubber17 committed:
Update stream-analytics-common-troubleshooting-issues.md

GitHubber17 committed:
Update stream-analytics-concepts-checkpoint-replay.md

barclayn committed:
changing from azure to az

Commit:
quickfix21

Commit:
Merge pull request #38070 from barclayn/kv-article-fix

changing from azure to az

Commit:
Merge pull request #38071 from MarkusVi/quickfix21

quickfix21

GitHubber17 committed:
Merge pull request #38050 from gatneil/master

fixed portal snapshot

GitHubber17 committed:
Merge pull request #38063 from cynthn/david-reassign

Reassigning Davids Windows topics

GitHubber17 committed:
Merge pull request #37633 from JasonWHowell/stream-analytics

Adding recovery information

GitHubber17 committed:
Merge pull request #38068 from MarkusVi/quickfix08

quickfix08

BrentL-Collabera committed:
Update marketplace-what-is-appsource.md

Fixed to have two H2 headings, and fixed title metadata.

GitHubber17 committed:
Merge pull request #38046 from JasonWHowell/patch-1

Update stream-analytics-introduction.md

martincoetzer committed:
Update azure-ad-choose-authn.md

Updating links

PRMerger15 committed:
Merge pull request #38077 from martincoetzer/patch-2

Update azure-ad-choose-authn.md

Commit:
Adding info for Backlogged events

Commit:
fix

Commit:
Merge pull request #38080 from JSeb225/patch-28

Add info on backlogged events

rmcmurray committed:
Updating dates and authors on articles

PRMerger20 committed:
Merge pull request #36979 from vladvino/patch-9

Edits to the quota and rate limit policies.

MSSedusch committed:
Merge pull request #1 from MicrosoftDocs/master

Update

rmcmurray committed:
Addressing Acrolinx suggestions

MSSedusch committed:
Workaround for SLES 12 SP3

Add workaround for targetcli package issue in SLES 12 SP3

PRMerger12 committed:
Merge pull request #38090 from MSSedusch/sap-sles-sp3

SAP: workaround for SLES 12 SP3

Commit:
support-matrix-change

Commit:
Merge pull request #38096 from sujayt/master

Support matrix change

Commit:
Merge pull request #38062 from MGoedtel/UpdateLA416

updated articles per PM feedback

Commit:
Edit.

PRMerger14 committed:
Merge pull request #38107 from jimdial/patch-153

Edit.

Commit:
Added question.

Commit:
Fixed formatting.

PRMerger18 committed:
Merge pull request #38108 from jimdial/patch-154

Added question.

Brenduns committed:
1803 update for post update instructions

v-geberr committed:
shorten toc name

v-geberr committed:
add links to apis

PRMerger15 committed:
Merge pull request #38109 from Brenduns/april-12

1803 update for post update instructions

douglaslMS committed:
Made updates from PM.

v-geberr committed:
seo

Commit:
Delete how-to-vscode-develop-azure-function.md

Commit:
Delete how-to-vscode-develop-csharp-module.md

itechedit committed:
Merge pull request #36924 from MRayermannMSFT/dev/marayerm/update-docs-for-1.0

Update Storage Explorer docs for next release

douglaslMS committed:
Added one row about Azure-SSIS IR per PM Sandy.

itechedit committed:
Merge pull request #37916 from msvijayn/vijayn-docs

Update ARM templates for Alerts with ActionGroup

itechedit committed:
Merge pull request #37975 from linda33wj/master

Fix SAP C4C copy source type in sample

itechedit committed:
Merge pull request #37981 from BrentL-Collabera/New_Azure_Marketplace_Docs

New Azure Marketplace docs describing AppSource

douglaslMS committed:
Fixed numbering issues in 2 articles per PR reviewer.

PRMerger17 committed:
Merge pull request #38114 from douglaslMS/adf-azure-limits

Added one row about Azure-SSIS IR per PM Sandy.

itechedit committed:
Merge pull request #37999 from stevestein/sql-db-link-fix

update repo link

ktoliver committed:
fix typo in redirects

itechedit committed:
Merge pull request #38057 from rimman/part_edits_part2

Corrected API name to SQL API

ktoliver committed:
fix redirect

bandersmsft committed:
cost mgt - updated h2 for clarity

ArLucaID committed:
Update overview.md

ArLucaID committed:
Update overview.md

tynevi committed:
Merge pull request #37709 from jeahyoun/jeahyoun-text-context

Create test context file.

PRMerger12 committed:
Merge pull request #38119 from bandersmsft/master

cost mgt - updated h2 for clarity

ktoliver committed:
Merge pull request #37756 from shizn/master

redirect develop and deploy doc

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ArLucaID committed:
Update known-issues.md

Commit:
PR comments

ArLucaID committed:
Update known-issues.md

ArLucaID committed:
Update how-to-use-vm-token.md

rmca14 committed:
Merge pull request #37675 from chetansriv/fluxxlabs

FluxxLabs tutorial update

ktoliver committed:
Merge pull request #37963 from douglaslMS/adf-ssis-activity

New article about SSIS Activity in ADF v2.

ktoliver committed:
Merge pull request #37990 from Minewiskan/patch-1

Update analysis-services-scale-out.md

rmca14 committed:
Merge pull request #37194 from srrengar/perfcounters

Perf counter changes

v-alje committed:
Merge pull request #38124 from MicrosoftDocs/FromPublicRepo

From public repo

rjagiewich committed:
Merge pull request #37413 from cephalin/tls

enforce tls

PRMerger19 committed:
Merge pull request #37918 from v-geberr/0416-phraselist-clarification

phrase list clarification

ktoliver committed:
Merge pull request #37947 from v-geberr/0416-http-response-codes

HTTP status codes

PRMerger20 committed:
Merge pull request #38120 from arluca/patch-9

Added IMDS clarifications

ktoliver committed:
Merge pull request #38082 from rmcmurray/20180411

Updating dates and authors on articles

rjagiewich committed:
Merge pull request #35869 from dksimpson/dean-405

Task 405: Create initial HTTP headers article

Commit:
Merge pull request #37997 from hpsin/patch-7

Minor cleanup
